packages/treasury/bundles/install-on-chain.js

```diff
@@ -123,12 +160,15 @@ export async function installOnChain({
 
   // Look up all the board IDs.
   const boardIdValue = [
-    ['INSTANCE_BOARD_ID', instance],
+    ['INSTANCE_BOARD_ID', treasuryInstance],
     ['INSTALLATION_BOARD_ID', stablecoinMachineInstall],
     ['RUN_ISSUER_BOARD_ID', centralIssuer],
     ['RUN_BRAND_BOARD_ID', centralBrand],
     ['AMM_INSTALLATION_BOARD_ID', autoswapInstall],
     ['LIQ_INSTALLATION_BOARD_ID', liquidationInstall],
+    ['BINARY_COUNTER_INSTALLATION_BOARD_ID', binaryCounterInstall],
+    ['COMMITTEE_INSTALLATION_BOARD_ID', committeeInstall],
+    ['CONTRACT_GOVERNOR_INSTALLATION_BOARD_ID', contractGovernorInstall],
```

Comment:
I'm struggling to judge whether these board / naming updates are correct. I suspect there are some external constraints... from dapp-treasury?

`installOnChain` has no doc comment.

vs-code is complaining that it can't find the `NameHub` type.

Reply:
I was mostly guessing here. I should ask @michaelfig  to review this PR and opine on the changes to `install-on-chain.js

Reply:
This list is specifically for the UI config that is returned to `dapp-treasury/ui`.  So, unless the Treasury UI is going to use those board IDs, please don't bother with them.

Rather than a 5-member committee that has no active members, I would prefer a "don't do anything except unilaterally replace my electorate" facet that could be handed to the bootstrap vat, so that the bootstrap could install the actual initial electorate.

It's fine if that's done in a future PR, but please explicitly mark all the inescapable committee governance in this file as a temporary hack just to allow the treasury not to crash during installation.

Reply:
The way governance works is that the electorate has to exist before the contract is instantiated, in order to make the governance visible to all. I'm fine with saying there's no governance during testNet, but the way I'd do that is by building a do-nothing electorate. I think that's straightforward, and may be useful in other, similar circumstances.

It would be visible that there's a place for governance, but no one can initiate governance actions in the current installation.